bliss/conf/base_config.yaml

```diff
@@ -188,7 +189,11 @@ surveys:
             # brick '3366m010' corresponds to SDSS RCF 94-1-12
             - ra: 336.6643042496718
               dec: -0.9316385797930247
-        bands: ${encoder.bands}
+        bands: [0, 1, 3] # grz
```

Comment:
Why is there a `3` here rather than a `2`?

Reply:
i.e., why wouldn't the bands be consecutive?

Reply:
Just because there is no i-band image for this brick

Reply:
Maybe we could remove `bands` from the config files, replacing it with a `use_multiband` binary setting. If true, we use all available bands. If false, we process only the reference band. Would that work?

Reply:
For debugging and benchmarking, it's still helpful be able to process single-band data, so I don't think we can switch entirely to multiband. But we'd never need to process single-band data that isn't the reference band. And we'd never need to process, say, just the odd numbered bands :-)

Reply:
This makes sense. Since this would be a refactoring change, I think it can be tracked by a new PR.